main.tf

```
@@ -1,20 +1,33 @@
 //TODO: Move the grafana module in the `modules/grafana` folder
 
+data "aws_region" "current" {
+  name = var.aws_region
+}
+
+data "aws_grafana_workspace" "this" {
+  workspace_id = module.managed_grafana.workspace_id
+}
+
+
 locals {
   #name        = local.locals_vars.locals.name
   description = "Amazon Managed Grafana workspace for ${var.name}"
 
-
   tags = {
     GithubRepo = "terraform-aws-observability-accelerator"
     GithubOrg  = "aws-observability"
   }
+
   grafana_iam_role_name = module.managed_grafana.workspace_iam_role_name
   iam_role_name         = "aws-observability-workspace-iam-role"
+  amg_ws_endpoint       = "https://${data.aws_grafana_workspace.this.endpoint}"
+  amg_ws_id             = module.managed_grafana.workspace_id
+
+  grafana_workspace_id = data.aws_grafana_workspace.this.workspace_id
 }
 
 module "managed_grafana" {
-  source = "terraform-aws-modules/managed-service-grafana/aws"
+  source = "github.com/liatrio/terraform-aws-managed-service-grafana.git"
```

I suggest spelling out this comment & ensuring that this repo is forkable + this works over HTTPS instead of SSH 
```suggestion
  # This is a fork of the upstream community edition and will be set back to the published
  # module version once the fix for adding `nac_configuration` is merged upstream.
  source = "github.com/liatrio/terraform-aws-managed-service-grafana.git"
```